Jenkinsfile

```diff
@@ -385,6 +385,60 @@ pipeline {
                         }
                     }
                 }
+             stage('cart-master-nightly') {
```

Comment:
This should be "Fault injection master" or similar, it's the fault injection tests.